hub/features/tests/test_serilize_deserialize.py

```diff
@@ -0,0 +1,46 @@
+from hub.features.features import Primitive,Tensor,FeatureDict 
```

Comment:
It would be better to create function test_something(), so we can have have multiple tests in single file later on. As bugs keeps pouring we will add them inevitably.